discovery-infra/tests/conftest.py

```diff
@@ -8,8 +8,13 @@
 
 qe_env = False
 
+
+def is_qe_env():
```

Comment:
why it is function? why not to set variable in env?

Reply:
Globals in Python are global to a module, not across all modules.
So instead of depending on a variable, and instead of creating a Singleton instance, I chose to have a function :)